src/acp.js

```diff
@@ -91,14 +91,26 @@ function handleGlobalBanDelete(user, data) {
 }
 
 function handleListUsers(user, data) {
-    var name = data.name;
-    if (typeof name !== "string") {
-        name = "";
+    var query = data.query;
```

Comment:
well I originally had it as `field`, but thought that was confusing as well. I'll change it to searchTerm if it pleases the crown.